libcxx/CMakeLists.txt

```diff
@@ -481,6 +481,21 @@ include(config-ix)
 include(HandleLibC) # Setup the C library flags
```

Comment:
General comment on the whole patch: Let's use `_LIBCPP_foo` instead of `LIBCXX_foo` for the macro names. That's what we do consistently everywhere, and we must use a reserved identifier.